It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

Set up the hypothesis test.
Write down all the steps followed for setting up the test.
Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

#### Null hypothesis: 
HHS sample's mean systolic blood pressure is the same as the population mean systolic blood pressure. 

In [1]:
# creating variables for the test
pop_mean = 120
pop_std = 21.21
n = 100
sample_mean = 130.1

In [2]:
# Calculating statistic by hand:
import math

statistic = (sample_mean - pop_mean)/(pop_std/math.sqrt(n))
print ("T-statistic calculated by hand is: ", round(statistic,4))

T-statistic calculated by hand is:  4.7619


In [6]:
# Calculating statistic in Python:

from scipy import stats
import numpy as np
# creating the sample: 
hhs_sample = np.random.normal(sample_mean, pop_std, n)
# t-test:
t_statistic, p_value = stats.ttest_1samp(hhs_sample, pop_mean)
print ("T-statistic from Python calculation is: ", t_statistic)
alpha = 0.05
if p_value < alpha: 
    print ("We can reject null hypothesis.")
else: 
    print ("We can not rejest the null hypothesis.")

T-statistic from Python calculation is:  5.452104973965612
We can reject null hypothesis.


#### Conclusion: 
Considering the calculated statistic of 4.7619 is higher than the z-value (1.984) and falls within the critical region, we can reject the null hypothesis. It means that with 95% of confidence HHS sample's mean systolic blood pressure is not the same as the population mean systolic blood pressure. 